io.greenfire.Greenery.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.greenfire.Greenery.desktop</id>
+  <name>Greenery</name>
+  <developer_name>Greenfire</developer_name>
+  <summary>Cryptocurrency bookkeeping and accounting wallet</summary>
+  <description>
+    <p>
+    The First Cryptocurrency Wallet with Bookkeeping and Accounting Functionality in mind for Avid traders, Small Business owners and Accountants
+    </p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://greenfire.io/</url>
+  <screenshots>
+    <screenshot type="default">https://raw.githubusercontent.com/GreenfireInc/greenfireinc.github.io/main/portfolio-large.png</screenshot>
```

Comment:
screensot URL must be inside an `image` tag.

Reply:
I have added image tags around all screenshots